components/Discussion/DiscussionProvider/components/DiscussionMetaHelper.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { inQuotes } from '@project-r/styleguide'
+
+import { useDiscussion } from '../context/DiscussionContext'
+import Meta from '../../../Frame/Meta'
+import { getFocusUrl } from '../../CommentLink'
+import { useTranslation } from '../../../../lib/withT'
+
+/**
+ * Render meta tags for a focused comment.
+ * @constructor
+ */
+const DiscussionMetaHelper = () => {
+  const { t } = useTranslation()
+  const { discussion, loading, error } = useDiscussion()
+
+  if (loading || error || !discussion) {
+    return null
+  }
+
+  if (discussion.comments.focus) {
+    const metaFocus = discussion.comments.focus
+
+    console.debug('render comment meta')
+
+    return (
+      <Meta
+        data={{
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore
+          title: t('discussion/meta/focus/title', {
+            authorName: metaFocus.displayAuthor.name,
+            quotedDiscussionTitle: inQuotes(discussion.title)
+          }),
+          description: metaFocus.preview,
+          url: getFocusUrl(discussion, discussion.comments.focus)
+        }}
+      />
+    )
+  }
+
+  if (discussion) {
+    console.debug('render discuission meta')
+    return (
+      <Meta
+        data={{
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore
+          title: t('discussion/meta/title', {
+            quotedDiscussionTitle: inQuotes(discussion.title)
+          }),
+          url: getFocusUrl(discussion)
+        }}
+      />
+    )
+  }
```

Comment:
you don't need to do this because the article page renders and controls the meta when not focussed